carbon-design-system/ibm-security
# fix(tagwall): accommodate truncation in tags and list items

## Affected issues

- Resolves #511 

## Proposed changes

- truncate tags in `TagWall` to 40ex per input from design
- truncate list items in `TagWall` to width of container
- update storybook demo to show what happens with long strings

## Testing instructions

- The `TagWall` demo has a long string now, so you can see how it appears when truncated. You can then remove the tag, returning it the list below, and see how it appears truncated in the list of options.
